include/eve/module/real/core/function/regular/generic/load.hpp

```diff
@@ -125,7 +125,7 @@ namespace eve::detail
                                 , eve::as<Pack> const& tgt, Ptr ptr
                                 ) noexcept
   {
-    if constexpr(sanitizers_are_on)
+    if constexpr(sanitizers_are_on && !kumi::product_type<Ptr>)
```

Comment:
Why is this change?

Reply:
I have to check cause I thought I had it removed